wynnset/tapestry-wp
# Containerize Application With Docker

Closes #530 
Adds:
 - Docker-compose server configuration for full wordpress stack
 - Dockerfile for building vue application

Steps to test:
 1. Download and install [Docker](https://docs.docker.com/get-docker/). The installation comes with the command line tools required.
 2. Create your `wp-config.php` in the Docker folder. The only changes that need to be made from the `wp-config-sample.php` currently there is generation of the secret keys. There are instructions to do so in the file. If you want, you can also just copy the keys from your current wordpress folder.
 3. Navigate to the directory and start the stack with `docker-compose up`
 4. Load `localhost:8000` in your browser and check if wordpress is loading
 5. Create a wordpress account and activate the plugins (Should be the same plugins installed on your local wordpress)
 6. Change your hyperlink settings to post-title
 7. Create a new Tapestry and view it in the browser. The app should hook in, and you should see the loading page.
